pom.xml

```diff
@@ -73,12 +74,14 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 			<version>${jackson.version}</version>
+			<scope>provided</scope>
```

Comment:
This needs to be excluded in the alfresco connector and not marked as provided here. This may break other users of this project.